### PR TITLE
Update/add decentab devices

### PR DESCRIPTION
**Pull request by decentlab**
#### Summary

Updated decentlab devices.

#### Changes

- updated decentlab logo
- updated dl-iam images
- added dl-pr26
- added dl-mbx
- added `pressure` into the `sensors` list in schema.json

### CODE REVIEW

File: lib/draft/schema.json [hunk 1]
```diff
@@ -793,6 +793,7 @@
               "moisture",
               "ph",
               "pir",
+              "pressure",
```
Comment (johanstokking) [inline lib/draft/schema.json:796]: This needs to be added to https://github.com/lorawan-schema/draft-devices

Comment (johanstokking) [reply lib/draft/schema.json:796]: I just did that, please rebase your changeset against `master`

File: vendor/decentlab/dl-iam-codec.yaml [hunk 2]
```diff
@@ -7,8 +7,8 @@ uplinkDecoder:
         bytes: [0x02, 0x0b, 0xbd, 0x00, 0x7f, 0x0b, 0x92, 0x6a, 0x51, 0x5d, 0x48, 0xbc, 0x4e, 0x02, 0x62, 0x00, 0x69, 0x81, 0xc7, 0x00, 0x00, 0x93, 0xd4, 0x00, 0x0b, 0x01, 0x11]
       output:
         data:
-          device_id: 3005
           protocol_version: 2
+          device_id: 3005
```
Comment (johanstokking) [inline vendor/decentlab/dl-iam-codec.yaml:11]: The order shouldn't matter; were you having issues?

**Review by johanstokking — COMMENTED** [review]
Awesome guys, minor comments only

File: lib/draft/schema.json (hunk 1 above)
Comment (decentlab) [reply lib/draft/schema.json]: The changeset has been rebased against `master`.

Just a side note:
- does it make sense to convert this to a git submodule?
- there's a tab character in front of `pressure` that can be fixed next time.

File: vendor/decentlab/dl-iam-codec.yaml (hunk 2 above)
Comment (decentlab) [reply vendor/decentlab/dl-iam-codec.yaml:11]: The order doesn't matter. It's just for future consistency. @jbiveroni

File: lib/draft/schema.json (hunk 1 above)
Comment (johanstokking) [reply lib/draft/schema.json]: > * does it make sense to convert this to a git submodule?

It is in fact a git subtree

> * there's a tab character in front of `pressure` that can be fixed next time.

Will fix next time, thanks

**Review by johanstokking — APPROVED** [review]

**Pull request merged**